addons/product/views/product_document_views.xml

```diff
@@ -107,6 +107,8 @@
         <field name="model">product.document</field>
         <field name="arch" type="xml">
             <tree editable="top" multi_edit="true">
+                <field name="res_model" column_invisible="True"/>
```

Comment:
why do you add this res_model field ? :thinking:

Reply:
Because I needed to check [here](https://github.com/odoo/odoo/pull/153465/files#:~:text=%3Cfield%20name%3D%22shown_on_product_page%22%20invisible%3D%22res_model%20%3D%3D%20%27product.product%27%22/%3E) if its a product or a variant in order to display the toggle `shown_on_product_page` and `res_model` isn't defined in list view

Reply:
Indeed, could have been defined next to their place of use, but it doesn't have to be changed :ok_hand:

Reply:
Noted for next time :ok_hand: